src/components/legend/handle_click.js

```diff
@@ -27,6 +28,8 @@ module.exports = function handleClick(g, gd, numClicks) {
     else if(numClicks === 2) mode = itemDoubleClick;
     if(!mode) return;
 
+    var toggleGroup = groupClick === undefined || groupClick === 'togglegroup';
```

Comment:
`groupclick` has a `dflt` value, so is there ever a case where we still get `undefined`?